gnomad_qc/v3/sample_qc/subpop_analysis.py

```diff
@@ -53,21 +54,30 @@ def compute_subpop_qc_mt(
         & ~release_ht.was_split
         & hl.is_snp(release_ht.alleles[0], release_ht.alleles[1])
         & hl.is_missing(lcr_intervals.ht()[release_ht.locus])
-    ).key_by("locus")
-
-    # Densify the MT and include only sites defined in the qc_sites HT
-    mt = mt.select_entries(
-        "END", GT=mt.LGT, adj=get_adj_expr(mt.LGT, mt.GQ, mt.DP, mt.LAD)
     )
 
-    logger.info(f"Checkpointing the QC sites HT of biallelic SNVs that are not in low-confidence regions and have a popmax above the specified minimum allele frequency of {min_popmax_af}.")
+    logger.info(
+        f"Checkpointing the QC sites HT of biallelic SNVs that are not in low-confidence regions and have a popmax above the specified minimum allele frequency of {min_popmax_af}."
+    )
     qc_sites = qc_sites.checkpoint(
         get_checkpoint_path("qc_sites"),
         overwrite=args.overwrite,
         _read_if_exists=not args.overwrite,
     )
 
-    mt = densify_sites(mt, qc_sites, hl.read_table(last_END_position.path))
+    logger.info("Converting MT to VDS...")
+    mt = mt.select_entries(
+        "END", "LA", "LGT", adj=get_adj_expr(mt.LGT, mt.GQ, mt.DP, mt.LAD)
+    )
+
+    vds = hl.vds.VariantDataset.from_merged_representation(mt)
+    vds = hl.vds.split_multi(vds)
```

Comment:
Do we need to do the split_multi if this is only on bi-allelics and derived from the same MT as the release HT though?